src/resolvers/savedItem.ts

```diff
@@ -13,7 +13,7 @@ export async function tags(
   args,
   context: IContext
 ): Promise<Tag[]> {
-  return context.models.tag.getBySaveId(parent.id);
+  return await context.models.tag.getBySaveId(parent.id);
```

Comment:
added this for my own readability - my understanding is having `await` or no in this workflow should not affect its child resolvers